src/main/java/pb360/model/embedded/AnnotationInside.java

```diff
@@ -0,0 +1,80 @@
+package pb360.model.embedded;
+
+import java.util.Collections;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AnnotationInside {
+	private String annotationStarts;	
+	@JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
+	private List<String> annotationContent;
+	private String annotationEnds;
+	private String annotations;
+
+	@JsonIgnoreProperties(ignoreUnknown = true)
+	public AnnotationInside(@JsonProperty("annotationStarts") String annotationStarts,
+			@JsonProperty("annotationContent") List<String> annotationContent,
+			@JsonProperty("annotationEnds") String annotationEnds,
+			@JsonProperty("annotation") String annotations) {
+		
+		if(annotations instanceof String)
```

Comment:
what means of this code?

Reply:
This code what to find out if annotations is 1 String or Array but it doesn't work so i delete it too